build.sbt

```diff
@@ -616,18 +667,17 @@ lazy val test = project
     fork in IntegrationTest := true,
     javaOptions in IntegrationTest += "-Xmx1G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
-    testOptions in IntegrationTest += Tests.Setup( () => root.base.getAbsolutePath + "/pull-binary-libs.sh" ! ),
```

Comment:
We could check for it in `Test.Setup` and log a warning if the JARs are missing. I don't think the whole build should fail. Only a handful of partest test cases still need `pull-binary-libs`. Everything else is resolved through Ivy in the sbt build.